packages/location-field/src/mocks/handlers.js

```diff
@@ -7,6 +7,7 @@ export default [
   rest.get(
     "https://ws-st.trimet.org/pelias/v1/autocomplete",
     (req, res, ctx) => {
+      // return res; /* note: remove the // to make autocomple work in the story */
```

Comment:
When I use this line instead of the currently uncommented one I get "could not reach geocoder". I'm open to re-adding support for a live geocoder, maybe via a new story?

Reply:
Hey Miles ... a new story would be great.
But maybe not for this PR...   Happy to delete my comment if need be.